src/Custom/VectorStores/GeneratorStubs.cs

```diff
@@ -0,0 +1,6 @@
+namespace OpenAI.VectorStores;
+
+/// <summary> The options to configure how <see cref="VectorStore"/> objects are retrieved and paginated. </summary>
+// Customization: fix namespace
+[CodeGenType("VectorStoreFileAssociationCollectionOptions")] public partial class VectorStoreFileAssociationCollectionOptions{ }
+[CodeGenType("VectorStoreFileAssociationOrder")] public partial struct VectorStoreFileAssociationOrder{ }
```

Comment:
This one should be called `VectorStoreFileAssociationCollectionOrder` for consistency. 

(It's a little bit of a mouthful for now, but I also have a proposal in the works to rename `VectorStoreFileAssociation*` to simply `VectorStoreFile*` to match the REST API, but this rename will come later in the future. 🙂)